Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java

```diff
@@ -45,6 +47,7 @@
 
     private static final String TAG = "Queue";
 
+    public List<String> sortedSongObjectIds;
     private Context context;
     private List<String> songObjectIds;
```

Comment:
Do you still need two list of songs, one list sorted, the other unsorted? Can you keep the list always sorted?

Reply:
The unsorted list is passed into the adapter from the activity and then I use the unsorted list to build the comparable model that helps me populate the sorted list.